lib/takeaway.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu'
+require_relative 'kitchen'
+require_relative 'text'
+
+class TakeAway
+  attr_reader :kitchen
+  def initialize
+    @menu = Menu.new
```

Comment:
put the dependencies as a parameter in the initialise method so you can use mocks/doubles in your rspec